src/Generator/ValueGenerator.php

```diff
@@ -343,7 +354,9 @@ public function generate()
                 $output .= $value;
                 break;
             case self::TYPE_ARRAY:
-                $output .= 'array(';
+            case self::TYPE_ARRAY_LONG:
+            case self::TYPE_ARRAY_SHORT:
+                $output .= ($type == self::TYPE_ARRAY_SHORT) ? '[' : 'array(';
```

Comment:
I am not sure if I understand you correctly. Are you talking about having $startArray and $closeArray vars instead of having the ?: structure inline?